src/routes/faithfulpride/+page.svelte.md

```diff
@@ -3,49 +3,48 @@ import Badge from '$lib/components/BadgeRaw.svelte';
-
-![Modrinth Downloads](https://img.shields.io/modrinth/dt/invalid?logo=modrinth&label=Modrinth%20downloads&color=%231bd96a&style=for-the-badge) ![CurseForge Downloads](https://img.shields.io/curseforge/dt/invalid?logo=curseforge&label=CurseForge%20Downloads&color=%23f16436&style=for-the-badge) ![GitHub Downloads (all assets, all releases)](https://img.shields.io/github/downloads/pridecraft-studios/faithfulpride/total?style=for-the-badge&logo=github&label=Github%20Downloads) ![GitHub commit activity](https://img.shields.io/github/commit-activity/t/pridecraft-studios/faithfulpride?style=for-the-badge&logo=github) 
-
 
-This is Faithful Pride. A LGBTQ+ Pride themed Resource Pack made by [Pridecraft Studios](https://git.pridecraft.gay/), with the 32x quality and looks of [Faithful](https://faithfulpack.net/). We make Minecraft queer.
-## Who
-We're [Pridecraft Studios](https://pridecraft.gay): A team that makes LGBT content for Minecraft. We're working on [PridePack](https://git.pridecraft.gay/PridePack), [PrideMod](https://git.pridecraft.gay/PrideMod), and a few other [projects](https://github.com/orgs/Pridecraft-Studios/repositories).
-[![ko-fi](https://rawcdn.githack.com/intergrav/devins-badges/1aec26abb75544baec37249f42008b2fcc0e731f/assets/cozy/donate/kofi-plural_vector.svg)](https://ko-fi.com/W7W4NLJWR)
-# Screenshots
-
-![A few biis pollinating a flower with glow berries on the background](https://cdn.modrinth.com/data/yPbBrzEX/images/f74e7f243502e23ec6f8805d38a4c6a5a06df292.png)
-![Six pride flag themed paitings on the walls of a Lush Cave](https://cdn.modrinth.com/data/yPbBrzEX/images/f1798d383e8654db8ad0f4f8702a8d2401740e1a.png)
-![A group of biis, many of the baby variant, pollinating an Azalea Bush.](https://cdn.modrinth.com/data/yPbBrzEX/images/117a6d9341620d97ba32b31c45677019e764d635.png)
-
-## How can I test new stuff out?
-You can join [our Discord](https://discord.pridecraft.gay) or follow [our Mastodon](https://tech.lgbt/@pridecraft) and get early features sneak-peaks, or check out the [GitHub Releases](https://git.pridecraft.gay/PridePack) for pre-releases. We'll post early unreleased content, such as early content reveals on [our Ko-Fi](https://ko-fi.com/W7W4NLJWR)
-
-## How can I contribute?
-You can help us translate Pridepack to more languages on
-[Crowdin](https://crowdin.com/project/pridepack/settings) [![Crowdin](https://badges.crowdin.net/pridepack/localized.svg)](https://crowdin.com/project/pridepack)
-all translations will be added to packs such as Faithful Pride and Pride Bones.
-You can also indirectly contribute by voting on polls on [the Discord](https://discord.pridecraft.gay), or [our Mastodon](https://tech.lgbt/@pridecraft)
+# Faithful Pride
 
+<aside class="shields">
+<ModrinthShield modid="faithfulpride"/>
+<CurseForgeShield modid="faithful-pride" type="texture-packs" projectid="831872"/>
```

Comment:
Update project ID when possible.

Reply:
It kinda needs to exist first sob

Reply:
Fair enough; I'll just zero it then or make it intentionally fail?

Reply:
Considering this out of scope.